src/Microsoft.SqlTools.SqlCore/Microsoft.SqlTools.SqlCore.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<AssemblyName>MicrosoftSqlToolsSqlCore</AssemblyName>
+		<Nullable>disable</Nullable>
+		<EnableDefaultItems>false</EnableDefaultItems>
+		<EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+		<EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
+		<EmbeddedResourceUseDependentUponConvention>false</EmbeddedResourceUseDependentUponConvention>
+		<EnableDefaultNoneItems>false</EnableDefaultNoneItems>
+		<DefineConstants>$(DefineConstants);NETCOREAPP1_0;TRACE</DefineConstants>
+		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+		<PreserveCompilationContext>true</PreserveCompilationContext>
+		<AssemblyTitle>SqlTools Hosting Library</AssemblyTitle>
+		<Description>Provides hosting services for SqlTools applications.</Description>
```

Comment:
These seem to be copied from the hosting project, please update with better values